src/toil/wdl/wdl_analysis.py

```diff
@@ -903,7 +903,11 @@ def parse_declaration_expressn_fncall(self, name, params, es):
                 if name.source_string == 'length':
                     es = es + 'len('
                 elif name.source_string == 'stdout':
-                    return es + 'stdout'
+                    # stdout is a str or bytes object. Since stdout() expects a File return
```

Comment:
I'm concerned that we don't know whether stdout is str or bytes. Are we sure the generated code is always going to know and use the right one?

Reply:
Valid concern - it seems like the captured stdout from both subprocess and the docker API should be in bytes, but one of the methods in the docker API suggests str is returned. I'll make sure to check the type and make the output consistent before writing to the file store.